docs/DeveloperGuide.md

```diff
@@ -144,9 +144,7 @@ The `Model` component,
 <puml src="diagrams/StorageClassDiagram.puml" width="550" />
```

Comment:
im not entirely sure how storage works but do write down some logical steps or examples for event storage. It seems like its only for person in the DG